core/closures.ml

```diff
@@ -363,8 +376,8 @@ struct
                treat them as globals *)
           let o, _ =
             List.fold_right
-              (fun (f, _, _, _, _) (o, fs) ->
-                 let o, f = o#binder f in
+              (fun fundef (o, fs) ->
```

Comment:
Maybe a bit late now, but for future for these sort of projections I think it is nicer to perform them by pattern matching; in this instance `{ binder; _ }`.

Reply:
I've done that in most other cases